src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -334,7 +334,7 @@ private static Process StartNewVisualStudioProcess(string installationPath, int
 
                 // HACK: 16.10P2 contains an LSP client bug where on solution closed, server activation tasks that are not already completed / cancelled
                 // do not properly get cancelled.  When a new solution is opened these incomplete server ativation tasks are not cleared.
-                // Any feature that waits for LSP server activations to complete will hang on the old incomplete server activation tasks.
+                // Any feature that waits for LSP server activations to complete will stop responding because of the old incomplete server activation tasks.
```

Comment:
Hang is explicitly allowed in this technical context. Replacement is acceptable with one of the following technical alternatives on condition that the correct alternative is known and used.

* **Deadlock** (e.g. lock inversion)
* **Livelock** (e.g. infinite loop)
* **UI delay** long enough that it is indistinguishable from one of the former

Reply:
For the scenario described here, 'deadlock' is an appropriate term. The internal issue is already resolved as Won't Fix, so it is at least as acceptable (if not more so) to just close this PR as won't fix.